ansible/roles_ocp_workloads/ocp4_workload_le_certs_cert_manager/tasks/pre_workload.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Implement your Pre Workload deployment tasks here
+- name: Ensure directory exists
+  file:
+    path: "/tmp/{{ guid }}"
+    state: directory
+
+- name: Install helm
+  include_tasks: install-helm.yaml
+
+- name: Install redhat-cop helm repository
+  command: helm repo add redhat-cop https://redhat-cop.github.io/helm-charts
+
+- name: Update helm repo
+  command: helm repo update
```

Comment:
There is a helm module in the Kubernetes.core collection. Use that instead of command. *if* you insist on doing helm.

Reply:
ack, will get that done